src/backend/PortabilityService.DependencyInjection/PortabilityService.DependencyInjection.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net461</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <NoWarn>$(NoWarn),SA1636</NoWarn>
```

Comment:
nit: replace with semi-colon

Reply:
Fixed

---
In reply to: [178096069](https://github.com/Microsoft/dotnet-apiport/pull/598#discussion_r178096069) [](ancestors = 178096069)